exercises/concept/meltdown-mitigation/.meta/exemplar.py

```diff
@@ -11,12 +11,8 @@ def is_criticality_balanced(temperature, neutrons_emitted):
     - The product of temperature and neutrons emitted per second less than 500000.
     """
     output = temperature * neutrons_emitted
-    balanced = False
 
-    if (temperature < 800 and neutrons_emitted > 500) and output < 500000:
-        balanced = True
-
-    return balanced
+    return temperature < 800 and neutrons_emitted > 500 and output < 500000
```

Comment:
As stated in the issue, this exercise is about `conditionals`, and every task should use or practice them - as contrived as that feels.  This task in the exemplar now displays only the use of Booleans.  If this task cannot be re-written to use/display the use of `conditionals`, then another task should be made or this task should be omitted from the exercise.

Reply:
As it is, the task is misleading, so 
> another task should be made or this task should be omitted from the exercise.
but I do not have the inspiration for this at the moment.